source/indexers/contracts/Indexers.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/utils/structs/EnumerableSet.sol";
+
+/**
+ * @title AirSwap: Indexer URL Registry
+ * @notice https://www.airswap.io/
+ */
+contract Indexers {
+  using SafeERC20 for IERC20;
+  using EnumerableSet for EnumerableSet.AddressSet;
+
+  EnumerableSet.AddressSet internal supportingStakers;
+  mapping(address => string) public stakerURLs;
+
+  event Stake(address indexed account);
+  event Unstake(address indexed account);
+  event SetURL(address indexed account, string url);
+
+  /**
+   * @notice Set the URL for a staker
+   * @param _url string value of the URL
+   */
+  function setURL(string calldata _url) external {
```

Comment:
Optionnal:
It adds an URL to the set, not setting one 
Proposal :   function addURL(string calldata _url) external {...}

Reply:
Good thought, though `setURL` "sets" the URL for `msg.sender` and there is symmetric no `removeURL` as the sender would just `setURL` with a blank string.